stdlib/3/tokenize.pyi

```diff
@@ -41,7 +41,7 @@ class Untokenizer:
 def untokenize(iterable: Iterable[_Token]) -> Any: ...
 def detect_encoding(readline: Callable[[], bytes]) -> Tuple[str, Sequence[bytes]]: ...
 def tokenize(readline: Callable[[], bytes]) -> Generator[TokenInfo, None, None]: ...
-def generate_tokens(readline: Callable[[], str]) -> Generator[TokenInfo, None, None]: ...
+def generate_tokens(readline: Callable[[], bytes]) -> Generator[TokenInfo, None, None]: ...  # undocumented
```

Comment:
That's incorrect, isn't it?
https://github.com/python/cpython/blob/825358ab80d0e17e287c1e36a91dece0a97cb219/Lib/tokenize.py#L612-L618

Reply:
Yes, confirmed in shell on 3.6. @srittau what was the basis for your change?

Reply:
I've noticed that there are difference with py27 in this regard, so maybe it got confused?

Reply:
No idea why I thought readline() should return bytes or why I even stumbled over this. I probably misread `_tokenize()`.